contrib/grimshot

```diff
@@ -13,26 +13,40 @@
 ## See `man 1 grimshot` or `grimshot usage` for further details.
 
 getTargetDirectory() {
-  test -f ${XDG_CONFIG_HOME:-~/.config}/user-dirs.dirs && \
-    . ${XDG_CONFIG_HOME:-~/.config}/user-dirs.dirs
+  test -f "${XDG_CONFIG_HOME:-~/.config}/user-dirs.dirs" && \
+    . "${XDG_CONFIG_HOME:-~/.config}/user-dirs.dirs"
 
-  echo ${XDG_SCREENSHOTS_DIR:-${XDG_PICTURES_DIR:-$HOME}}
+  echo "${XDG_SCREENSHOTS_DIR:-${XDG_PICTURES_DIR:-$HOME}}"
 }
 
-if [ "$1" = "--notify" ]; then
-  NOTIFY=yes
-  shift 1
-else
-  NOTIFY=no
-fi
+NOTIFY=
```

Comment:
You're setting notify to an empty string here, but the [existing] check below is `"$NOTIFY" = "no"`.

Running `grimshot --cursor save active` showed a notification, when it shouldn't.

Reply:
oops